.github/workflows/nx.template.yml

```diff
@@ -57,7 +58,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{ inputs.nx-head }}
```

Comment:
Why is is this ref removed?

Reply:
It's to use the default configuration. I'll move this in a different PR to test it in isolation